.github/workflows/full-test.yml

```diff
@@ -19,9 +19,9 @@ jobs:
         run: sudo apt-get update && sudo apt-get install make
 
       - name: Setup Go
-        uses: actions/setup-go@v3
+        uses: actions/setup-go@v4
         with:
-          go-version: 1.19
+          go-version: "1.20.x"
```

Comment:
why not just `1.20` ?

Reply:
> why not just `1.20` ?

there was an issue with that, i forgot which exactly, but it either didn't get the latest or not worked at all